packages/wells/src/client/api/wellboresApi.ts

```diff
@@ -145,4 +146,40 @@ export class WellboresAPI {
 
     return measurements?.items.map((measurement: Measurement) => this.addLazyMethodsForMeasurement(measurement)) 
   }
+
+  public getCasings = async (wellOrWellboreId: number): Promise<Sequence[] | undefined> => {
```

Comment:
Could we not add a casing object in the same way we have for trajectories, which then allows the getCasingData to be a method on that object?

Reply:
Could we also add this into the `addLazyMethodForWellbore` method so we can directly get casings from the wellbore object

Reply:
I'll add it :+1: